src/Banner.js

```diff
@@ -91,7 +91,7 @@ Banner.propTypes = {
   iconName: PropTypes.string,
   onClose: PropTypes.func,
   showIcon: PropTypes.bool,
-  text: PropTypes.string,
+  text: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
```

Comment:
`PropTypes.node` should include text nodes (strings) as well IIRC

Reply:
update to just use node